### PR TITLE
Fix #1312 - Removed models "Affiliation" and "UserAffilition" 

### LINKED ISSUE #1312: Remove models from Accounts app
**Expected Behaviour** : Remove the `Affiliation` and `UserAffiliation` model from the `Accounts` app.
**Deliverables:**
- [ ] Remove the `Affiliation` and `UserAffiliation` model from accounts app.
- [ ] Create and name the migration.

### PR DESCRIPTION
**Pull request by markslingerland**
+ all it's references

Issue #1312

### CODE REVIEW

File: apps/accounts/migrations/0001_initial.py [hunk 1]
```diff
@@ -17,31 +17,6 @@ class Migration(migrations.Migration):
```
Comment (RishabhJain2018) [inline apps/accounts/migrations/0001_initial.py]: Please create a new migration after removing the models.

File: apps/accounts/models.py [hunk 2]
```diff
@@ -33,36 +33,12 @@ class Meta:
...
-        db_table = 'user_affiliation'
-
-
 class Profile(TimeStampedModel):
     """
     Model to store profile of a user
     """
     user = models.OneToOneField(User)
     contact_number = models.CharField(max_length=10, blank=False, null=True)
-    affiliation = models.CharField(max_length=512)
```
Comment (RishabhJain2018) [inline apps/accounts/models.py]: Please do not remove this field.

**Comment by RishabhJain2018** [comment]
Hey @markslingerland Please see review comments :)

**Comment by deshraj** [comment]
@markslingerland Please let us know if you are still working on this otherwise we can close this.

**Comment by HargovindArora** [comment]
If it's inactive then can I work on this?

**Comment by deshraj** [comment]
@HargovindArora sure, go ahead. Closing this one.

**Pull request closed without merging**